ui/guest/src/store/actions/index.ts

```diff
@@ -17,3 +17,19 @@
 import { createAction } from '@reduxjs/toolkit';
 
 export const contentReady = /*#__PURE__*/ createAction('content_ready');
```

Comment:
Pretty sure there are a few actions not here (e.g. the ones prefixed by `document:`)